src/sentry/integrations/github/integration.py

```diff
@@ -215,21 +215,29 @@ def get_commit_context(
         if not lineno:
             return None
         try:
-            blame_range = self.get_blame_for_file(repo, filepath, ref, lineno)
+            blame_range: Sequence[Mapping[str, Any]] | None = self.get_blame_for_file(
+                repo, filepath, ref, lineno
+            )
+
+            if blame_range is None:
+                return None
         except ApiError as e:
             raise e
 
         try:
-            commit = max(
+            commit: Mapping[str, Any] = max(
                 (
                     blame
-                    for blame in blame_range
+                    for blame in blame_range or ()
```

Comment:
Does it need the `or ()` since it's returning early if it's None?

Reply:
Right, I think I added some redundant check since the typing was so strict lol.